storage-proofs/src/drgraph.rs

```diff
@@ -1,15 +1,24 @@
 use std::cmp;
 use std::marker::PhantomData;
 
+#[cfg(feature = "disk-trees")]
+use merkle_light::merkle::next_pow2;
 use rand::{ChaChaRng, OsRng, Rng, SeedableRng};
 use rayon::prelude::*;
+#[cfg(feature = "disk-trees")]
+use std::path::Path;
+// FIXME: Figure out what to do with all the conditional `use`es.
 
 use crate::error::*;
 use crate::hasher::pedersen::PedersenHasher;
 use crate::hasher::{Domain, Hasher};
+#[cfg(feature = "disk-trees")]
+use crate::merkle::DiskMmapStore;
 use crate::merkle::MerkleTree;
 use crate::parameter_cache::ParameterSetIdentifier;
 use crate::util::{data_at_node, NODE_SIZE};
+#[cfg(feature = "disk-trees")]
+use crate::SP_LOG;
```

Comment:
Maybe group all the `cfg`s together, so it is easier to understand, what that pulls in

Reply:
I got the idea that `rustfmt` was pulling them apart (respecting import name order), I'll try to group them with braces.

Reply:
It seems Rust doesn't like grouping imports with `{...}`, were you thinking in a particular arrangement?

Reply:
I just meant like this:

```rust
#[cfg(feature = "disk-trees")]
use a;
#[cfg(feature = "disk-trees")]
use b;

use other;
use d;
```